source/lac/slepc_solver.cc

```diff
@@ -195,22 +195,17 @@ namespace SLEPcWrappers
       for (unsigned int i = 0; i < *n_converged; i++)
         {
           double residual_norm_i = 0.0;
-          // EPSComputeResidualNorm is L2-norm and is not consistent with the stopping criteria
-          // used during the solution process.
-          // Yet, this is the norm which gives error bounds (Saad, 1992, ch3):
-          //   | \lambda - \widehat\lambda | <= ||r||_2
-          ierr = EPSComputeResidualNorm (eps, i, &residual_norm_i);
-
-          // EPSComputeRelativeError may not be consistent with the stopping criteria
```

Comment:
i would keep those comments in case someone later will have a question why we chose the function below.

Reply:
Fair point. I'll go back and add these in (they do need a slight rewrite since EPSComputeRelativeError is deprecated).